programs/vote/src/vote_instruction.rs

```diff
@@ -87,21 +88,24 @@ pub enum VoteInstruction {
     Withdraw(u64),
 
     /// Update the vote account's validator identity (node id)
-    ///    requires authorized voter signature
+    ///    requires authorized voter and new validator identity signature
     ///
-    /// Expects 2 Accounts:
+    /// Expects 3 Accounts:
     ///    0 - Vote account to be updated with the Pubkey for authorization
-    ///    1 - Clock sysvar Account that carries clock bank epoch
+    ///    1 - New validator identity (node id)
+    ///    2 - Clock sysvar Account that carries clock bank epoch
     ///
-    UpdateNode(Pubkey),
```

Comment:
This is a breaking change but doesn't matter because any previous UpdateNode(Pubkey) instructions will fail regardless due to the extra signature check